### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by modernNeo**
adds more direction to both devs and bot_managers

### CODE REVIEW

File: documentation/Working_on_the_Bot.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+1. `.deleterole <whatever_name_you_want_to_use>`  
+1. `.iamn <whatever_name_you_want_to_use>`  
+1. `.whois <whatever_name_you_want_to_use>`  
+1. `.deleterole <whatever_name_you_want_to_use>`  
+1. `.roles`  
+1. `.Roles`  
+
+ ## Reporting Issues  
+
+ If you come across issues that googling wont solve, you can email the bot-managers with the details at `csss-bot-manager@sfu.ca`. Please note that if your email is not detailed enough, the bots may not necessarily respond. Please over-provide rather than under-provide.
```
Comment (TitanVJ) [inline documentation/Working_on_the_Bot.md]: Absolutely HARAM. We(At least not me) are not here to teach ppl how to python and bot, googling will solve and/or #projects-and-dev on our discord(maybe this'll bring the channel back from the dead).  At the very least I dont wanna trouble shoot over email, in person is much better.

Comment (modernNeo) [reply documentation/Working_on_the_Bot.md]: >Absolutely HARAM
Does that mean really good or really bad?

Comment (TitanVJ) [reply documentation/Working_on_the_Bot.md]: bad

Comment (modernNeo) [reply documentation/Working_on_the_Bot.md]: what would you change it to?

Comment (TitanVJ) [reply documentation/Working_on_the_Bot.md]: maybe say after googling first try out discord server and ask around in projects and dev channel, if that fails and only then as a last resort email the bot managers.

**Comment by modernNeo** [comment]
@TitanVJ there ya go

**Pull request merged**